core/src/main/java/org/elasticsearch/rest/action/admin/cluster/node/hotthreads/RestNodesHotThreadsAction.java

```diff
@@ -40,19 +38,17 @@
 /**
  */
 public class RestNodesHotThreadsAction extends BaseRestHandler {
+    public RestNodesHotThreadsAction(RestGlobalContext context) {
+        super(context);
+        context.getController().registerHandler(RestRequest.Method.GET, "/_cluster/nodes/hotthreads", this);
```

Comment:
I just pushed a patch along those lines. Its not what you asked for but its something....